lib/classes/PluginManager.js

```diff
@@ -73,7 +73,7 @@ class PluginManager {
   }
 
   loadServicePlugins(servicePlugs) {
-    const servicePlugins = (typeof servicePlugs !== 'undefined' ? servicePlugs : []);
+    const servicePlugins = (servicePlugs && typeof servicePlugs !== 'undefined' ? servicePlugs : []);
```

Comment:
Yes, I changed it before seeing this.  Good idea checking if it's an array, but I don't think it should throw an error.  It's fine if the `plugins` property is empty.